docs/pim/attributes/date_and_time.md

```diff
@@ -1,68 +1,16 @@
 ---
 description: Date and time attribute type allows you to store product information related to time, like an expiration date or date of manufacturing.
-edition: lts-update
 ---
 
 # Date and time attributes
 
 The date and time [attribute type](products.md#product-attributes) allows you to represent date and time values as part of the product specification in the [Product Information Management](pim_guide.md) system.
 
-It's released as an [LTS update](release_process_and_roadmap.md) that you can install on top of your current project to expand the capabilities of your [[= product_name =]], starting with version v4.6.17 and higher.
-
 You can use it to store, for example, manufacturing dates, expiration dates, or event dates, all with specified accuracy.
 
-
-## Installation
-
-### Download the bundle
-
-To get the most recent stable version of this package, open a command terminal, navigate to your project directory, and run the following command:
-
-``` bash
-composer require ibexa/product-catalog-date-time-attribute
-```
-
-### Enable the bundle
```

Comment:
This must be covered in the update doc - if someone didn't use LTS updates in v5, they must set them up when upgrading